client/src/components/HostViews/HostMixin.js

```diff
@@ -32,6 +32,10 @@ export default {
       type: Boolean,
       required: true
     },
+    numOfSpots: {
```

Comment:
yes. I will change it to be more explicit

Reply:
I changed it to numOfPlayerSpots as that number corresponds to how many spots are allowed to be occupied, not how many players are in the room. I made this variable as it can give devs/hosts more flexibility to adjust this number instead of having it as a hard coded value. The reason for this prop is because the UI design in figma calls for an "Open Spot" card to let people know that there is an open spot in the room that they can take. <img width="604" alt="Screen Shot 2022-11-05 at 2 50 01 PM" src="https://user-images.githubusercontent.com/76519301/200136174-b838939e-bf8d-48df-959e-79e46a615ebb.png">